README.md

```diff
@@ -11,15 +11,14 @@ Exposes the following, all of which require the logging config object:
 
 The config object to be passed to each service has the following properties:
 
+- LOGGING_THRESHOLD
+  - trace/debug/info/warn/error (default = info) - the threshold at which to create log entries
 - LOGGING_TYPE (can be local|remote|none)
   - local: log to local console only
   - remote: log to local console and remote endpoint
   - none: perform no logging
 - REMOTE_LOGGING_ENDPOINT: the full URL to the endpoint where log messages are sent via AJAX POST
 - REMOTE_ERROR_REPORT_ENDPOINT: the full URL to the endpoint where user error reports will be sent via AJAX POST
-- LOGGING_LEVEL (can be debug|error)
-  - debug: log both debug and error messages
-  - error: only log error messages (debug messages are not logged)
```

Comment:
I think it's worth adding a change log file to highlight that this has changed. Also - isn't this a breaking change worthy of a V2 denotion? If a consumer of this package has a liberal version policy, they will get a surprise here.